src/pybind/mgr/cephadm/module.py

```diff
@@ -928,7 +928,8 @@ def extra_ceph_conf(self) -> 'CephadmOrchestrator.ExtraCephConf':
         try:
             j = json.loads(data)
         except ValueError:
-            self.log.exception('unable to laod extra_ceph_conf')
+            msg = 'Unable to load extra_ceph_conf: Cannot decode JSON: %s' % (e)
+            self.log.exception('%s: \'%s\'' % (msg, data))
```

Comment:
1. there is no `e` in this context, as we're doing `except ValueError:` instead of `except ValueError as e:`
2. [`self.log.exception`](https://docs.python.org/3/library/logging.html#logging.Logger.exception) already prints the exception. thus there is no need to add the message of the exception.

```suggestion
            msg = 'Unable to load extra_ceph_conf: Cannot decode JSON'
            self.log.exception('%s: \'%s\'', msg, data)
```

Reply:
agree, adding the exception message is redundant